docs/content/usage/scenarios/custom-scenario/attack.md

```diff
@@ -16,7 +16,7 @@ The Infection Monkey can simulate various [ATT&CK](https://attack.mitre.org/matr
 - **ATT&CK matrix** You can use the ATT&CK configuration section to select which techniques you want the Infection Monkey to simulate.
 For the full simulation, use the default settings.
 - **Exploits -> Credentials** This configuration value will be used for brute-forcing. The Infection Monkey uses the most popular default passwords and usernames, but feel free to adjust it according to the default passwords common in your network. Keep in mind a longer list means longer scanning times.
-- **Network -> Scope** Disable “Local network scan” and instead provide specific network ranges in the “Scan target list”.
+- **Network -> Scope** Disable “Local network scan” and instead provide specific network ranges in the “Scan target list.”
```

Comment:
@ybasford You've got more experience in tech writing than I do, so feel free to disagree and correct me.

In technical matters, quotations are very specific. The place where you can specify network ranges is called "Scan target list", it has no period as part of its name.